README.md

```diff
@@ -77,51 +74,40 @@ PARAMETERS = {
     "schema": "demo_data"
 };
 
-SELECT * FROM data_source.car_sales;
-
--- Step 2: Create a skill
-CREATE SKILL my_skill
-USING
-    type = 'text2sql',
-    database = 'data_source',
-    tables = ['car_sales'],
-    description = 'car sales data of different car types';
-
-SHOW SKILLS;
-
--- Step 3: Deploy a conversational model
-CREATE ML_ENGINE langchain_engine
-FROM langchain
-USING
-      openai_api_key = 'your openai-api-key';
-      
-CREATE MODEL my_conv_model
-PREDICT answer
-USING
-    engine = 'langchain_engine',
-    model_name = 'gpt-4',
-    mode = 'conversational',
-    user_column = 'question',
-    assistant_column = 'answer',
-    max_tokens = 100,
-    temperature = 0,
-    verbose = True,
-    prompt_template = 'Answer the user input in a helpful way';
-
-DESCRIBE my_conv_model;
-
--- Step 4: Create an agent
-CREATE AGENT my_agent
-USING
-    model = 'my_conv_model',
-    skills = ['my_skill'];
-
-SHOW AGENTS;
-
--- Step 5: Query an agent
-SELECT * FROM my_agent WHERE question = 'What is the average price of cars from 2018?';
+-- See some of the data in there
+SELECT * FROM demo_postgres_db.car_sales;
+
 ```
 
+Now you can create an egent that can answer questions over unstructued information in this database (let's use the Python SDK)
+
+```python
+import mindsdb_sdk
+
+# connects to the default port (47334) on localhost 
+server = mindsdb_sdk.connect()
+
+# create an agent (lets create one that can answer questions over car_sales table
+agent = server.agents.create('my_agent')
+agent.add_database('demo_postgres_db', ['car_sales'], 'car sales data')
+# you can do it for all tables as follows:
```

Comment:
Each `add_database` produce new skill, so for single connection (`demo_postgres_db`) better to use it one time. Description is related to skill itself. May be better will be:
```python
agent.add_database(
    database='demo_postgres_db',
    tables=['car_sales'],
    description='The table "car_sales" contains car sales data')
```

Reply:
fixed